src/components/Attachment/Attachment.js

```diff
@@ -147,7 +159,7 @@ class Attachment extends PureComponent {
             key={`key-video-${a.id}`}
           >
             <div className="str-chat__player-wrapper">
-              <ReactPlayer
```

Comment:
ReactPlayer being used anywhere else in this component or is that an unused import at the top now?